packages/libant/scripts/common.sh

```
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 SCRIPT_DIR="$(cd "$(dirname "$0")/.." && pwd)"
-ROOT_DIR="$(cd "$SCRIPT_DIR/.." && pwd)"
+ROOT_DIR="$(cd "$SCRIPT_DIR/../.." && pwd)"
```

Shellcheck (0.11.0)

[warning] ROOT_DIR appears unused. Verify use (or export if used externally).

(SC2034)